### PR TITLE
Gtk3-merging features

### PR DESCRIPTION
**Pull request by morrolinux**
* CTRL+b to toggle **bold** when in text editor 
* CTRL+ALT+Plus/Minus to change font size on the fly (while in text editor)
* Changed "graph" journal style color form blue to grey (might be useful to give user customization just like background color)
* Go to next/prev. page with right/left arrows (really useful when reading slides, could be nice if customizable by user as well in the future)
* Save a timestamp for each taken stroke when recording, and audio playback feature (audio mapping)

### CODE REVIEW

File: src/control/xojfile/LoadHandler.cpp [hunk 1]
```diff
@@ -697,6 +711,12 @@ void LoadHandler::parserEndElement(GMarkupParseContext* context, const gchar* el
 		handler->pos = PARSER_POS_IN_PAGE;
 		handler->layer = NULL;
 	}
+	else if (handler->pos == PARSER_POS_IN_LAYER && strcmp(element_name, "timestamp") == 0)
+	{
+		printf("parserEndElement: timestamp reached - LEVEL: %d\n",handler->pos);
```
Comment (andreasb242) [inline src/control/xojfile/LoadHandler.cpp]: Debug code, should be removed

File: src/control/xml/XmlPointNode.cpp [hunk 2]
```diff
@@ -33,6 +55,24 @@ void XmlPointNode::writeOut(OutputStream* out)
...
 
+	/**
+	 * Write timestamp before each stroke.
+	 * This way, when reading, we only need to store 
+	 * 1 timestamp value at a time 
+	 * and assign it to the consequent stroke.
+	 * By adding it this way we don't break 
+	 * xournal's fileformat backcompatibility
+	 */
+	out->write("<timestamp ");
```
Comment (andreasb242) [inline src/control/xml/XmlPointNode.cpp]: May there should be an option to completely disable timestamps

File: src/gui/MainWindow.cpp [hunk 3]
```diff
@@ -29,6 +29,11 @@ using std::cout;
 using std::endl;
 namespace bf = boost::filesystem;
 
+//those two have to be moved somewhere else in the future
```
Comment (andreasb242) [inline src/gui/MainWindow.cpp]: Generally: Add TODO so they are highlighted

File: src/gui/XournalView.cpp [hunk 4]
```diff
@@ -278,24 +278,24 @@ bool XournalView::onKeyPressEvent(GdkEventKey* event)
 
 	if (event->keyval == GDK_KEY_Left)
 	{
-		if (control->getSettings()->isPresentationMode())
-		{
+		//if (control->getSettings()->isPresentationMode())
```
Comment (andreasb242) [inline src/gui/XournalView.cpp]: Why comment out?

File: toolbar.ini [hunk 5]
```diff
@@ -1,69 +0,0 @@
-# Xournal++ Toolbar configuration
```
Comment (andreasb242) [inline toolbar.ini]: Why did you delete this, is this not the main toolbar config?

File: src/gui/XournalView.cpp (hunk 4 above)
Comment (morrolinux) [reply src/gui/XournalView.cpp]: It was functional to me, for skipping to the next/prev slide with just the right/left arrow key, without having to be in presentation mode. I guess the best thing to to would be to make it customizable

Comment (andreasb242) [reply src/gui/XournalView.cpp]: I didn't ever use the presentation mode, therefore I'm asking...

File: src/control/xml/XmlPointNode.cpp (hunk 2 above)
Comment (morrolinux) [reply src/control/xml/XmlPointNode.cpp]: Adding a check to write/load them only if a recording has been taken

**Comment by morrolinux** [comment]
It compiles!!!
